catalogue_pipeline/transformer/src/main/scala/uk/ac/wellcome/transformer/modules/UnidentifiedWorkKeyPrefixGeneratorModule.scala

```diff
@@ -0,0 +1,21 @@
+package uk.ac.wellcome.transformer.modules
+
+import com.google.inject.Provides
+import com.twitter.inject.TwitterModule
+import javax.inject.Singleton
+import uk.ac.wellcome.models.work.internal.UnidentifiedWork
+import uk.ac.wellcome.storage.s3.KeyPrefixGenerator
+
+object UnidentifiedWorkKeyPrefixGeneratorModule extends TwitterModule {
+  @Provides
+  @Singleton
+  def provideKeyPrefixGenerator(): KeyPrefixGenerator[UnidentifiedWork] =
+    new UnidentifiedWorkKeyPrefixGenerator()
+}
+
+class UnidentifiedWorkKeyPrefixGenerator
```

Comment:
What's the difference between this and the one defined earlier in the diff?

Reply:
The other one is for Json. The idminter doesn't know about the Work structure

Reply:
It is incorrectly named though, will fix thanks